ert_gui/ertwidgets/summarypanel.py

```diff
@@ -103,3 +108,18 @@ def addColumn(self, text):
         layout.addStretch(1)
 
         self.layout.addLayout(layout)
+
+
+def _runlength_encode_list(strings: List[str]) -> List[Tuple[str, int]]:
+    """Runlength encode a list of strings.
+
+    Returns a list of tuples, first element is the string, and the second
+    element is the count of consecutive occurences of the string at the current
+    position."""
+    string_counts: List[Tuple[str, int]] = []
```

Comment:
Out of curiosity, why not to use `Dict[str, int]`? i.e., `If string in string_counts: string_counts[string]+=1 else string_counts[string]=0` ?